marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/schemas/ExternalMetadata.java

```diff
@@ -0,0 +1,71 @@
+

```

Comment:
Can you actually exclude this and any other schema-generated classes? I submitted a separate PR to update them.

Reply:
Removed!